imagebuildah/build_notlinux.go

```diff
@@ -0,0 +1,19 @@
+//go:build !linux
+
+package imagebuildah
+
+import (
+	"github.com/containers/buildah/define"
+	"github.com/containers/storage"
+)
+
+// platformSetupContextDirectoryOverlay() may set up an overlay _over_ the
+// build context directory, and sorts out labeling.  Returns either the new
+// location which should be used as the base build context or the old location;
```

Comment:
Same as https://github.com/containers/buildah/pull/5975/files#r1981811503 I think comment needs to be fixed.